src/backend/joanie/tests/test_settings.py

```diff
@@ -0,0 +1,38 @@
+"""
+Test suite for Joanie's main module
+"""
+from unittest import mock
+
+from django.test import TestCase
+
+from joanie.settings import get_release
+
+
+# pylint: disable=unused-argument
+class GetReleaseTestCase(TestCase):
+    """Battle test the settings.get_release function"""
+
+    @mock.patch("builtins.open", side_effect=FileNotFoundError)
+    def test_returns_default_without_version_file(self, *args):
```

Comment:
According to the handle book, it should be named:
```suggestion
    def test_settings_get_release_returns_default_without_version_file(self, *args):
```

Reply:
Hmm I'm not quit sure its useful to include in each test name the module name but ok to include the method name
`test_get_release_returns_default_without_version_file`

Reply:
I use it a lot in richie to run tests for a module:
```
bin/pytest -k test_settings
```

Reply:
![AAAAH!](https://media.giphy.com/media/26ufdipQqU2lhNA4g/giphy.gif)

Reply:
<img src="https://media.giphy.com/media/dVheWqLaSIhyWk8fOH/giphy.gif">